website/siteConfig.js

```diff
@@ -96,6 +96,11 @@ const users = [
     image: 'img/workforce-logo.png',
     infoLink: 'https://www.workforce.com/',
   },
+  {
+    caption: 'Tines',
+    image: 'img/tines-logo.svg',
+    infoLink: 'https://www.tines.com',
+  },
```

Comment:
Would you mind placing this bit between TaskRabbit and TriumphPay to preserve alphabetical order?